src/project-root.ts

```diff
@@ -33,13 +33,13 @@ export function excludePattern(moreExcludes: string[] = []) {
   return `**/{${[...moreExcludes, ...config.getConfig().projectRootsSearchExclude].join(',')}}`;
 }
 
-export async function findClosestProjectRootPath(candidatePaths?: string[]) {
+export async function findClosestProjectRootPath(candidatePaths?: vscode.Uri[]) {
```

Comment:
I think all these functions should either be named `...Uri`, or maybe just skip the prefix (if the latter, `findClosestProjectRoot` in this case).

Reply:
Yes I agree that the names are incorrect. I'll change them.